README.md

```diff
@@ -64,6 +67,17 @@ Once in the virtual environment, use the command `pipenv install PACKAGE_NAME1 P
 
 **NOTE:** Do not install packages without pipenv. We will not be supporting requirements.txt standard. 
 
+#### For Windows 
+
+[Download](https://www.seleniumhq.org/download/) the web driver according to version/type of your browser. Make sure that EXE file downloaded is inside the the project folder. In your development environment you can run the py file.
+
+#### For Linux
+
+Follow this command in your development Environment in your project folder
+```
+$ sudo apt-get install python-pip
+$ sudo pip install selenium
```

Comment:
I don't think these instructions are enough as I got an error that chromedriver could not be found. Can you please look into this?